docs/gethue/content/en/posts/2021-11-15-create-sql-tables-from-excel-files.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Create SQL tables from excel files
+author: Hue Team
+type: post
+date: 2021-11-15T00:00:00+00:00
+url: /blog/2021-11-15-create-sql-tables-from-execl-files
+sf_thumbnail_type:
+  - none
+sf_thumbnail_link_type:
+  - link_to_post
+sf_detail_type:
+  - none
+sf_page_title:
+  - 1
+sf_page_title_style:
+  - standard
+sf_no_breadcrumbs:
+  - 1
+sf_page_title_bg:
+  - none
+sf_page_title_text_style:
+  - light
+sf_background_image_size:
+  - cover
+sf_social_sharing:
+  - 1
+sf_related_articles:
+  - 1
+sf_sidebar_config:
+  - left-sidebar
+sf_left_sidebar:
+  - Sidebar-2
+sf_right_sidebar:
+  - Sidebar-1
+sf_caption_position:
+  - caption-right
+sf_remove_promo_bar:
+  - 1
+ampforwp-amp-on-off:
+  - default
+categories:
+  - Version 4.11
+  - Development
+  - Query
+
+---
+
+If you are a Hue user then you might be familiar with the Hue [Direct Importer](/blog/2021-05-26-improved-hue-importer-select-a-file-choose-a-dialect-create-a-table/). It lets you create tables from a local CSV file. Now, you will be able to create SQL table from excel(.xlsx) file also.
```

Comment:
Please replace "Now, you will be able to create SQL table from excel(.xlsx) file also." with "Now you can also create SQL tables by importing Excel (.xlsx) files into Hue."

Reply:
Done, 
Thanks